parsons/utilities/json_format.py

```diff
@@ -11,3 +11,32 @@ def arg_format(arg):
     arg_list[0] = arg_list[0].lower()
 
     return ''.join(arg_list)
+
+
+def list_to_string(list_arg):
+    """
+    Convert a list object to a string and not raise error if None
+    """
+
+    if list_arg:
+        return ','.join(list_arg)
+    else:
+        return None
+
+
+def remove_empty_keys(dirty_dict):
+    """
+    Remove empty keys from a dictionary. This method is useful when passing jsons
+    in which a null field will update the value to null and you don't want that.
+    """
+
+    clean_dict = {}
+
+    for k, v in dirty_dict.items():
+        if v:
+            if str and len == 0:
```

Comment:
This line seems like a typo? It will always evaluate to `False` since `len == 0` is always `False` (`len` evaluates as a built-in function).

If you meant `isinstance(v, str) and len(v) == 0`, this will still always evaluate to `False`, since you filter out all empty strings on line 36 (since in Python, empty strings are falsy).

So, I think lines 37 + 38 are unnecessary.